Documentation/conventions.md

```diff
@@ -294,7 +294,24 @@ To add material to the meeting (let's say `https://example.com/agenda-meeting.ht
 
 ## Dokieli Document
 
-A 'Dokieli Document' is an HTML document with some linked-data annotations, but otherwise just HTML. So the 'Dokieli Document' tool does not store data in triples in RDF sources like most other tools do, but instead allows you to run an online HTML editor right on your pod. When you click 'Save' in the Dokieli editor, the HTML document is written to your pod using a http PUT request, and in that sense this editor makes use of the LDP (read-write web) functionalities of your pod. You can also use Dokieli as a third-party app, on https://dokie.li.
+[dokieli](https://dokie.li/) is a clientside editor for decentralised article publishing, annotations and social interactions. Source code is available from https://github.com/linkeddata/dokieli .
+
+Dokieli allows content creators to generate human- and machine-readable documents in HTML+RDFa, optionally embedding "data blocks" using the `<script type="">` mechanism to include content in Turtle, JSON-LD, and TriG. Dokieli also employs mechanisms to use media types that are acceptable by a server, e.g., JSON-LD, Turtle, HTML+RDFa, when for example generating data for servers conforming to the [Solid Protocol](https://solidproject.org/ED/protocol), [Linked Data Platform](https://www.w3.org/TR/ldp/), [Linked Data Notifications](https://www.w3.org/TR/ldn/), [Web Annotation Protocol](https://www.w3.org/TR/annotation-protocol), or [ActivityPub](https://www.w3.org/TR/activitypub/).
```

Comment:
Will dokieli magically support embedded YAML-LD data blocks upon finalization of that spec? Or will you need to adjust your code? (Please consider this a "heads-up" that YAML-LD is coming, if you weren't already aware.)
```suggestion
dokieli allows content creators to generate human- and machine-readable documents in HTML+RDFa, optionally embedding "data blocks" using the `<script type="">` mechanism to include content in Turtle, JSON-LD, and TriG. dokieli also employs mechanisms to use media types that are acceptable by a server (e.g., JSON-LD, Turtle, HTML+RDFa) when, for example, generating data for servers conforming to the [Solid Protocol](https://solidproject.org/ED/protocol), [Linked Data Platform](https://www.w3.org/TR/ldp/), [Linked Data Notifications](https://www.w3.org/TR/ldn/), [Web Annotation Protocol](https://www.w3.org/TR/annotation-protocol), or [ActivityPub](https://www.w3.org/TR/activitypub/).
```